donate/core/models.py

```diff
@@ -53,7 +53,12 @@ class DonationPage(Page):
 
     @cached_property
     def currencies(self):
-        return deepcopy(constants.CURRENCIES)
+        currencies = deepcopy(constants.CURRENCIES)
+        # Re-order currency `single` and `monthly` amounts
+        for currency in currencies:
+            currencies[currency]['presets']['single'].sort()
+            currencies[currency]['presets']['monthly'].sort()
+        return currencies
```

Comment:
This works, but it would probably be better if we sort `constants.CURRENCIES` when that gets loaded in, rather than resorting everytime `.currencies()` gets called?